.buildkite/scripts/steps/package_testing/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+.buildkite/scripts/bootstrap.sh
+
+node scripts/build --all-platforms --debug
```

Comment:
small nit, 
![image](https://user-images.githubusercontent.com/3143860/148820813-ac75e785-9734-4368-b0ee-f60ba9c828db.png)

```suggestion
echo "--- Build Kibana Distribution"
node scripts/build --all-platforms --debug
```